src/btree/bt_curnext.c

```diff
@@ -941,10 +941,12 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
 
 done:
 err:
-    if (total_skipped < 100)
-        WT_STAT_CONN_DATA_INCR(session, cursor_next_skip_lt_100);
-    else
-        WT_STAT_CONN_DATA_INCR(session, cursor_next_skip_ge_100);
+    if (total_skipped != 0) {
```

Comment:
Make this && instead of having the two ifs.

Reply:
Combining into `if(total_skipped != 0 && total_skipped < 100) ` would mean that an `else if (total_skipped >= 100) ` would be needed to ensure that the `cursor_next_skip_ge_100` statistic doesn't get incremented when `total_skipped = 0` which is why I left it as is.